vars/buildXamarinIos.groovy

```diff
@@ -0,0 +1,35 @@
+def call(Map config) {
+
+    String solution = config.solution ?: findSolution()
+    if (solutionName == null) {
+    	failBuildWithError("No solution found. Please specify a solution explictly or ensure you are top level directory.")
+    }
+
+    String configuration = config.configuration ?: "Release"
+    String platform = config.platform ?: "iPhone"
+    boolean buildIpa = config.buildIpa ?: true
+    boolean archive = config.archive ?: true
+
+    build(solution, configuration, platform, buildIpa, archive)
+}
+
+private def build(String solution, String configuration, String platform, boolean buildIpa, boolean archive) {
+	withCredentials([string(credentialsId: "FASTLANE_KEYCHAIN_PASSWORD", variable: "KEYCHAIN_PASSWORD")]) {
+        def keychain = "/Users/instil/Library/Keychains/fastlane.keychain-db"
```

Comment:
Remove the explicit instil user and use the `HOME` env var.
```
def keychain = "$HOME/Library/Keychains/fastlane.keychain-db"
```

Reply:
done